From: Konstantin Demin <rockdrilla@gmail.com>
Date: Tue, 27 Dec 2022 14:23:29 +0100
Subject: Use local (offline) fonts in html docs

Use Liberation font family (with fallback to DejaVu)
instead of (online via Web) Noto.

[jak: Just keep the fonts as set, we'll fall back to user fonts
 if they don't have noto installed, which is good]

Forwarded: no
---
 docs/styles/dark.css | 2 --
 1 file changed, 2 deletions(-)

diff --git a/docs/styles/dark.css b/docs/styles/dark.css
index 8f7bd67..e9a2ed1 100644
--- a/docs/styles/dark.css
+++ b/docs/styles/dark.css
@@ -1,5 +1,3 @@
-@import url(https://fonts.googleapis.com/css?family=Noto+Sans);
-
 /* Asciidoctor default stylesheet | MIT License | http://asciidoctor.org */
 /* Uncomment @import statement below to use as custom stylesheet */
 /*@import "https://fonts.googleapis.com/css?family=Open+Sans:300,300italic,400,400italic,600,600italic%7CNoto+Serif:400,400italic,700,700italic%7CDroid+Sans+Mono:400,700";*/
